src/config_legacy.cpp

```diff
@@ -398,6 +398,7 @@ namespace ConfigLegacy
         uint32_t customThemeR3Pressed;
         uint32_t customThemeA1Pressed;
         uint32_t customThemeA2Pressed;
+        uint32_t customThemeCooldownTimeInMs;        
```

Comment:
We also don't touch anything in config_legacy.cpp, this is kind of an internal knowledge thing so super easy to miss. You can remove any changes to config_legacy.cpp